src/js/index.js

```diff
@@ -54,18 +54,35 @@ const tabSwitch = async function (state,tab){
             markup[i].classList = [];
             markup[i].childNodes[1].classList = ['tab'];
         }
+        
 
-        //above selected change
-        markup[1 + (parseInt(tab.parentNode.id) - 1) * 2].classList.add('above-selected');
-        markup[1 + (parseInt(tab.parentNode.id) - 1) * 2].childNodes[1].classList.add('special-tab-1');
+        if (state.user.isSenior){
+            document.querySelector("#opp-decision").parentNode.parentNode.classList.add("invisible");
+            if (tab.parentNode.id == "1"){
+                markup[1].classList.add('above-selected');
+                markup[1].childNodes[1].classList.add('special-tab-1');
+                tab.parentNode.classList.add('selected-tab');
+                markup[7].classList.add('below-selected');
+                markup[7].childNodes[1].classList.add('special-tab-2');
+            } else if (tab.parentNode.id == "3"){
+                markup[3].classList.add('above-selected');
+                markup[3].childNodes[1].classList.add('special-tab-1');
+                tab.parentNode.classList.add('selected-tab');
+                markup[9].classList.add('below-selected');
+                markup[9].childNodes[1].classList.add('special-tab-2');
+            }
+        } else {
+            //above selected change
+            markup[1 + (parseInt(tab.parentNode.id) - 1) * 2].classList.add('above-selected');
+            markup[1 + (parseInt(tab.parentNode.id) - 1) * 2].childNodes[1].classList.add('special-tab-1');
 
-        //selected tab change
-        tab.parentNode.classList.add('selected-tab');
+            //selected tab change
+            tab.parentNode.classList.add('selected-tab');
 
-        //below selected change
-        markup[1 + (parseInt(tab.parentNode.id) + 1) * 2].classList.add('below-selected');
-        markup[1 + (parseInt(tab.parentNode.id) + 1) * 2].childNodes[1].classList.add('special-tab-2');
-        console.log("tab highlighed");
+            //below selected change
+            markup[1 + (parseInt(tab.parentNode.id) + 1) * 2].classList.add('below-selected');
+            markup[1 + (parseInt(tab.parentNode.id) + 1) * 2].childNodes[1].classList.add('special-tab-2');
+        }        
     }
```

Comment:
Can we make this more understandable with comments or refactoring

Reply:
Done

Reply:
I know it's UI changes, it doesn't tell me exactly what's going on though

Reply:
I added some extra comments about what UI changes here do. "[1 + (parseInt(tab.parentNode.id) + 1) * 2]" here is to find the correct index of the above and bellow selected tabs(it's complicated because there are text in the childNodes of the navigator).